src/js/pages/Themes.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { withRouteData } from 'react-static';
+
+import Hero from 'js/modules/Hero';
+import TileGroup from 'js/modules/TileGroup';
+
+const Themes = ({ allThemes }) => {
+  const links = allThemes.edges.map(theme => ({
```

Comment:
we have a very similar method in src/js/helpers/cleanData.js --> cleanServiceLinks
maybe we can make that method more generic and share?

Reply:
i tried that at first but I felt like I was passing too many params around then got lazy. Logged it as a tech debt issue so its on the radar to refactor https://github.com/cityofaustin/techstack/issues/244